packages/docusaurus-theme-classic/src/nprogress.css

```diff
@@ -16,7 +16,7 @@
 }
 
 #nprogress .bar {
-  background: #29d;
+  background: var(--ifm-color-primary);
```

Comment:
We can now use the primary color instead of a hardcode color for the progress bar!